pkgs/build-support/agda/default.nix

```diff
@@ -40,13 +40,14 @@ let
         allPackages = withPackages (filter self.lib.isUnbrokenAgdaPackage (attrValues self));
       };
     };
-    inherit (Agda) meta;
+    # Agda is a split package with multiple outputs; do not inherit them here.
+    meta = removeAttrs Agda.meta [ "outputsToInstall" ];
```

Comment:
The interaction of multiple-output packages and `outputsToInstall` was very surprising to me (Why does changing a _meta_ attribute cause an evaluation error?? I believe I ran into #255947). Is removing `outputsToInstall` for the derived package the right thing to do here?